notebooks/catvton/catvton.ipynb

```diff
@@ -18,14 +18,18 @@
     "Teaser image from [CatVTON GitHub](https://github.com/Zheng-Chong/CatVTON)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        demo.launch(debug=True, share=True)
</pre><p>please remove share=True in this line (in this case, there is no difference between try and except conditions)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2519/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>